ui/src/app/cohort-search/overview/overview.component.html

```diff
@@ -55,41 +31,87 @@ <h2 class="count-header" id="total-count">
+        (click)="navigateTo('review')"
+        [disabled]="(isRequesting$ | async) || !cohort"
+        [popper]="'Review participant level data'"
+        [popperTrigger]="'hover'">
+        <clr-icon shape="copy" class="is-solid disabled" size="30"></clr-icon>
+      </button>
     </div>
+    <h2 class="count-header" id="total-count">
+      Total Count:
+      <span *ngIf="(isRequesting$ | async)" class="spinner spinner-sm">
+        Loading...
+      </span>
+      <span class="count-header" *ngIf="!(isRequesting$ | async) && (total$ | async) !== -1">
+        {{ total$ | async | number }}
+      </span>
+      <span *ngIf="(error$ | async) && !temporal.flag">
+        <clr-icon
+          [popper]="errorPopper"
+          [popperTrigger]="'hover'"
+          shape="error-standard"
+          size="18"></clr-icon>
+      </span>
+      <span *ngIf="!(isRequesting$ | async) && ((temporal?.flag && temporal.tempLength && (total$ | async) === null) || (total$ | async) === -1)">
+        - -
+        <clr-icon
+          [popper]="errorPopper"
+          [popperTrigger]="'hover'"
+          shape="warning-standard"
+          size="18"></clr-icon>
+      </span>
+    </h2>
   </div>
-<ng-container *ngIf="(total$ | async) > 0 && !(isRequesting$ | async)">
-  <ng-container *ngIf="chartData$ | async as chartData">
-    <div class="card bg-faded">
-      <div class="card-header header-text">
-        Results by Gender
-      </div>
-      <div *ngIf="showGenderChart" class="card-block">
-        <app-gender-chart *ngIf="chartData.size"
-          [data]="chartData">
-        </app-gender-chart>
-      </div>
+<div *ngIf="(total$ | async) > 0 && !(isRequesting$ | async) && chartData$ | async as chartData">
+  <div class="card bg-faded">
+    <div class="card-header header-text">
+      Results by Gender
     </div>
-    <div class="card bg-faded">
-      <div class="card-header header-text">
-        Results By Gender, Age Range, and Race
-        <clr-icon shape="sort-by"
-          [class.is-info]="stackChart"
-          (click)="toggleChartMode()">
-        </clr-icon>
-        <!--<clr-icon shape="angle"-->
-          <!--[dir]="showComboChart ? 'up' : 'down'"-->
-          <!--(click)="toggleShowCombo()">-->
-        <!--</clr-icon>-->
-      </div>
-      <div *ngIf="showComboChart" class="card-block">
-        <app-combo-chart *ngIf="chartData.size"
-          [data]="chartData"
-          [mode]="stackChart ? 'stacked' : 'normalized'">
-        </app-combo-chart>
-      </div>
+    <div *ngIf="showGenderChart" class="card-block">
+      <app-gender-chart *ngIf="chartData.size"
+        [data]="chartData">
+      </app-gender-chart>
+    </div>
+  </div>
+  <div class="card bg-faded">
+    <div class="card-header header-text">
+      Results By Gender, Age Range, and Race
+      <clr-icon shape="sort-by"
+        [class.is-info]="stackChart"
+        (click)="toggleChartMode()">
+      </clr-icon>
+      <!--<clr-icon shape="angle"-->
```

Comment:
Do you still need this commented out code? (Just making sure its still giving value.)

Reply:
AFAIK we don't need it, removed